web_poet/page_inputs/response.py

```diff
@@ -0,0 +1,26 @@
+from typing import Union
+
+import attrs
+
+from web_poet.page_inputs.browser import BrowserResponse
+from web_poet.page_inputs.http import HttpResponse
+from web_poet.page_inputs.url import ResponseUrl
+
+
+@attrs.define
+class HttpOrBrowserResponse:
```

Comment:
Could you please add selectors support to this dependency?

Reply:
Good one! https://github.com/scrapinghub/web-poet/pull/195/commits/1021a11be8247007279ea08065d32f49844d45df